### DISCUSSION TITLE
How to add another Cloud provider

### DISCUSSION THREAD

**Discussion by strawbs1** [skypilot-org/skypilot#2526] [General]
Hi we are looking at adding support for SkyPilot to our Cloud, we are onboarding a large number of GPUs in the coming months:
https://www.cudocompute.com/

Is there any documentation for writing a provider ?

**Comment by concretevitamin** [comment]
Hello @JungleCatSW! Yes, check out our latest documentation for adding a new cloud into SkyPilot here: https://github.com/skypilot-org/new-cloud-skeleton/pull/1 (h/t @justinmerrell).

**Reply by justinmerrell** [reply]
Glad to see this being put to use :) 

Let me get those changes in the review done so it is easier to follow.
